src/main/java/com/squareup/kotlinpoet/AnnotationSpec.kt

```diff
@@ -20,6 +20,7 @@ import java.io.IOException
 import java.io.StringWriter
 import java.lang.reflect.Array
 import java.util.Arrays
+import java.util.Locale
```

Comment:
Sure. Or just do a PR if you have time.

I think long-term we'll have a core KotlinPoet and then platform-specific helpers for things like java.lang.Class, TypeMirror, TypeElement, Filer, etc. in a kotlinpoet-jvm artifact.